nncf/common/tensor_statistics/statistic_point.py

```diff
@@ -63,11 +63,12 @@ def add_statistic_point(self, statistic_point: StatisticPoint) -> None:
                             _statistic_point.algorithm_to_tensor_collectors[algorithm].extend(
                                 statistic_point.algorithm_to_tensor_collectors[algorithm]
                             )
-                            return
-                        _statistic_point.algorithm_to_tensor_collectors[
-                            algorithm
-                        ] = statistic_point.algorithm_to_tensor_collectors[algorithm]
-                        return
+                        else:
+                            _statistic_point.algorithm_to_tensor_collectors[
+                                algorithm
+                            ] = statistic_point.algorithm_to_tensor_collectors[algorithm]
+                    return
+
```

Comment:
These changes are required because the following code snippet does not work on the current develop branch
```python
algo_0 = PostTrainingQuantization(preset=nncf.QuantizationPreset.MIXED, fast_bias_correction=True)
algo_1 = PostTrainingQuantization(preset=nncf.QuantizationPreset.MIXED, fast_bias_correction=False)

sa = OVStatisticsAggregator(calibration_dataset)
sa.register_statistic_points(algo_0.get_statistic_points(model))
sa.register_statistic_points(algo_1.get_statistic_points(model))
sa.collect_statistics(model)

m_0 = algo_0.apply(model, sa.statistic_points)
m_1 = algo_1.apply(model, sa.statistic_points)
```